outputs.tf

```diff
@@ -22,16 +22,9 @@ resource "random_id" "redpanda" {
   }
 }
 
-output "redpanda_map" {
-  value = { for i in range(length(aws_instance.redpanda[*].id)) :
-    lower(random_id.redpanda[i].b64_url) => aws_instance.redpanda[i].private_ip
-  }
-  description = "A map of random IDs to public IPs for the Redpanda instances."
-}
-
```

Comment:
not sure I understand the removal of the outputs.

Reply:
it was something i was testing. zero value add so i went a different route